lib/tasks/refinery/elasticsearch.rake

```diff
@@ -15,6 +15,9 @@ namespace :refinery do
 
     desc 'Recreate Elasticsearch index'
     task recreate: :setup_logger do
+      # Use the right language
+      ::I18n.locale = ::I18n.default_locale
```

Comment:
Indexing multiple locales is surprisingly hard. Basically you'd have to index multiple versions of the same document. Not supported now, but might be in the future.